CMakeLists.txt

```diff
@@ -277,7 +274,7 @@ if(ENABLE_CURVE)
       endif()
     else()
       message(
-        WARNING
+        FATAL_ERROR
```

Comment:
This makes no sense, the message says tweetnacl is going to be used instead, but it's now a fatal error

Reply:
With this option set to **ON**:
`option(WITH_LIBSODIUM "Use libsodium instead of built-in tweetnacl" ON)`
I would expect to see a clear error message, not a warning!

Reply:
no, it's intentional to keep things working